src/operator/contrib/dgl_graph.cc

```diff
@@ -24,6 +24,9 @@
 #include <mxnet/operator_util.h>
```

Comment:
@zheng-da could you help review this in #17984?

Reply:
DGL no longer depends on the contrib graph operators. I would suggest to deprecate this sampling operator and even remove it in MXNet 2.0. Please see my comment here. https://github.com/apache/incubator-mxnet/issues/16167#issuecomment-613160299